src/views/map.c

```diff
@@ -2179,6 +2180,55 @@ static gboolean _view_map_remove_pin(const dt_view_t *view, OsmGpsMapImage *pin)
   return osm_gps_map_image_remove(lib->map, pin);
 }
 
+static void track_add_point(OsmGpsMapTrack *track, OsmGpsMapPoint *point, OsmGpsMapPoint *prev_point)
```

Comment:
track_add_point => _track_add_point to indicate this is a local routine.

Reply:
done